app/src/main/java/org/gdg/frisbee/android/chapter/NewsFragment.java

```diff
@@ -92,20 +88,20 @@ public void onPreExecute() {
                     }
                 })
                 .setOnBackgroundExecuteListener(
-                    new CommonAsyncTask.OnBackgroundExecuteListener<String, ActivityFeed>() {
+                    new CommonAsyncTask.OnBackgroundExecuteListener<String, Activities>() {
                         @Override
-                        public ActivityFeed doInBackground(String... params) {
-                            ActivityFeed feed =
-                                (ActivityFeed) App.getInstance().getModelCache().get(Const.CACHE_KEY_NEWS + params[0]);
+                        public Activities doInBackground(String... params) {
+                            Activities feed =
+                                (Activities) App.getInstance().getModelCache().get(Const.CACHE_KEY_NEWS + params[0]);
```

Comment:
Since the classes in the cache are now changed, we shouldn't do this. 
We can either change the keys here. That will give us new space in cache. Or we can delete the cache on app update.